db/migrate/20210224120024_add_merged_into_to_patient.rb

```diff
@@ -0,0 +1,6 @@
+class AddMergedIntoToPatient < ActiveRecord::Migration[5.2]
+  def change
+    add_column :patients, :merged_into, :uuid
```

Comment:
Consider `merged_into_patient_id`? I like the idea of the foreign key having the `<model>_id` suffix. 

Additionally, shall we set up the associations necessary to say things like `patient.merged_to_patient` and `patient.merged_from_patients`?

Reply:
Hmm... I feel like we can do better with the naming here too 🤔  I'm thinking something like `patient.duplicate_patients`, and `patient.original_patient`, but that's not _quite_ right. It's not an original, it's a _new_ one that is now the source of truth/consolidated result. Are there better words?

Reply:
`merged_to_patient` and `merged_from_patients` seem like good names actually.

Reply:
`merged_into_patient_id` seems good. I thought about the associations, and figured we'll add them when we need them. I suspect we might not actually need that in code. Only in one-off analytics perhaps.